README.md

```diff
@@ -94,35 +93,35 @@ Run your application:
 node example.js
 ```
 
-Check out [systeminfo](https://github.com/GoogleChromeLabs/carlo/tree/master/examples/systeminfo) and [terminal](https://github.com/GoogleChromeLabs/carlo/tree/master/examples/terminal) examples with richer UI and RPC-based communication between the Web and Node under the [examples](https://github.com/GoogleChromeLabs/carlo/tree/master/examples) folder.
+Check out [systeminfo](https://github.com/GoogleChromeLabs/carlo/tree/master/examples/systeminfo) and [terminal](https://github.com/GoogleChromeLabs/carlo/tree/master/examples/terminal) examples with richer UI and RPC-based communication between the Web and Node in the [examples](https://github.com/GoogleChromeLabs/carlo/tree/master/examples) folder.
 
 <!-- [END getstarted] -->
 
 ## API
 
-Check out [API](https://github.com/GoogleChromeLabs/carlo/blob/master/API.md) to get familiar with Carlo API.
+Check out the [API](https://github.com/GoogleChromeLabs/carlo/blob/master/API.md) to get familiar with Carlo.
 
 
 ## Contributing to Carlo
 
-Check out [contributing guide](https://github.com/GoogleChromeLabs/carlo/blob/master/CONTRIBUTING.md) to get an overview of Carlo development.
+Look at the [contributing guide](https://github.com/GoogleChromeLabs/carlo/blob/master/CONTRIBUTING.md) to get an overview of Carlo's development.
 
 <!-- [START faq] -->
 
 ## FAQ
 
-#### Q: What was the motivation behind this project when we already have Electron and NW.js? How this differs from these platforms, how it helps to achieve something that's not possible/harder with these two?
+#### Q: What was the motivation behind this project when we already have Electron and NW.js? How does this project differ from these platforms, how does it achieve something that is not possible/harder with Electron or NW.js?
 
-- One of the motivations is to demonstrate how browser installed locally can be used with Node out of the box.
-- Unlike with Electron, Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability of the independent updates of the underlying components. Carlo is less about branding and is more about productivity + giving the control over bundling to the user.
+- One of the motivations of this project is to demonstrate how browsers that are installed locally can be used with Node out of the box.
+- Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability to independently update underlying components. Carlo gives the user control over bundling and is more about productivity than branding.
```

Comment:
'more about productivity than branding' was referring to Electron, now that Electron is out of the context, it no longer makes sense.